tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/DataStreamsMonitoringRabbitMQTests.cs

```diff
@@ -12,78 +12,89 @@
 using Datadog.Trace.TestHelpers.DataStreamsMonitoring;
 using FluentAssertions;
 using FluentAssertions.Execution;
-using ICSharpCode.Decompiler.Util;
 using VerifyXunit;
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Datadog.Trace.ClrProfiler.IntegrationTests;
-
-[UsesVerify]
-[Trait("RequiresDockerDependency", "true")]
-public class DataStreamsMonitoringRabbitMQTests : TestHelper
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
```

Comment:
Nitpicky

Changing from the top level namespace (no `{`) to the old way where there are `{` `}` makes the changes here in this file difficult to see.